### PR TITLE
Update default timeout and log_uri print for rs uri

### PR DESCRIPTION
**Pull request by dnil**
### This PR adds | fixes:
Occasional timeouts when using loqus client. The default timeout is more than 1000 times lower than default for other mongoDB components, so likely time to increase it quite a bit.

**How to prepare for test**:
- [ ] `ssh` to ...
- [ ] Install on stage:
`bash servers/resources/SERVER.scilifelab.se/update-[THIS_TOOL]-stage.sh [THIS-BRANCH-NAME]`

### How to test:
-

### Expected outcome:
- [ ]

### Review:
- [ ] Code approved by
- [ ] Tests executed by
- [ ] "Merge and deploy" approved by

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: mongo_adapter/client.py [hunk 1]
```diff
@@ -43,7 +43,7 @@ def check_connection(client):
 
 
 def get_client(host='localhost', port=27017, username=None, password=None,
-              uri=None, timeout=20, authdb=None):
+              uri=None, timeout=30000, authdb=None):
```
Comment (northwestwitch) [inline mongo_adapter/client.py:46]: 👍🏻

File: mongo_adapter/client.py [hunk 2]
```diff
@@ -76,8 +76,8 @@ def get_client(host='localhost', port=27017, username=None, password=None,
     # Will raise pymongo.errors.InvalidURI if incorrect uri
     uri_info = uri_parser.parse_uri(uri)
     # nodelist is a list of tuples with (<host>,<port>)
-    host = uri_info['nodelist'][0][0]
-    port = uri_info['nodelist'][0][1]
+    host_port = ",".join(["".join([str(item[0]), ":", str(item[1])]) for item in uri_info['nodelist']])
```
Comment (northwestwitch) [inline mongo_adapter/client.py:79]: Ah good, the connection string printed by log_uri was incomplete!

**Review by northwestwitch — APPROVED** [review]
Looks good!

Good to merge, but I'm wondering how will it work now from loqusdb, that is using this software to connect. Do we have to release this to PyPI? On the long run would be better to have loqus to connect independently to MongoDB, but it can be done in another moment!

**Comment by dnil** [comment]
Ok, that looks promising!

**Comment by northwestwitch** [comment]
We need to deploy the new version of this repo now, right?

**Pull request merged**